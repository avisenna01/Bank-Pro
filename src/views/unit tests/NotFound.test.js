import React from 'react'
import ReactDOM from 'react-dom';
import NotFound from "../NotFound";
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';

const rendercomponent = () => (shallow(<NotFound />));

describe('NotFound component try 1', () => {
    it('snapshot renders', () => {
        const renderedcomponent = rendercomponent();
        expect(renderedcomponent.find('div').length).toEqual(1);
    });
});

describe("NotFound component try 2", () => {
    test("snapshot renders", () => {
        const button = renderer.create(<NotFound />);
        expect(button.toJSON()).toMatchSnapshot();
    });
});

it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<NotFound />, div);
});