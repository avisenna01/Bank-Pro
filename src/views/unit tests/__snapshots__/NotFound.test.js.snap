// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound component try 2 snapshot renders 1`] = `
<div
  style={
    Object {
      "textAlign": "center",
    }
  }
>
  <h1>
    404
  </h1>
  <p>
    Not Found
  </p>
</div>
`;
